build.sbt

```diff
@@ -81,7 +81,7 @@ val customMergeStrategy: String => MergeStrategy = {
         MergeStrategy.filterDistinctLines
       case _ => MergeStrategy.deduplicate
     }
-  case "asm-license.txt" | "overview.html" =>
+  case "asm-license.txt" | "overview.html" | "cobertura.properties" =>
```

Comment:
Yup.

Here are the contents of the two current `cobertura.properties` files we're discarding and not merging, etc.:

```bash
unzip -p \
  ~/.ivy2/cache/com.google.gcloud/gcloud-java-core/jars/gcloud-java-core-0.0.9.jar \
  cobertura.properties
```
```
#Generated by maven-cobertura-plugin for project com.google.gcloud:gcloud-java-core:jar:0.0.9
#Fri Oct 09 00:08:23 UTC 2015
net.sourceforge.cobertura.datafile=/home/travis/build/GoogleCloudPlatform/gcloud-java/gcloud-java-core/target/cobertura/cobertura.ser
```

versus

```
unzip -p \
  ~/.ivy2/cache/com.google.gcloud/gcloud-java/jars/gcloud-java-0.0.9.jar \
  cobertura.properties
```
```
#Generated by maven-cobertura-plugin for project com.google.gcloud:gcloud-java:jar:0.0.9
#Fri Oct 09 00:09:20 UTC 2015
net.sourceforge.cobertura.datafile=/home/travis/build/GoogleCloudPlatform/gcloud-java/gcloud-java/target/cobertura/cobertura.ser
```